diff --git a/src/js/constants/defaults.js b/src/js/constants/defaults.js
index 2f7e9fa8..24d8b32b 100644
--- a/src/js/constants/defaults.js
+++ b/src/js/constants/defaults.js
@@ -2,13 +2,13 @@ import { coreCitationStyles } from '../../../data/citation-styles-data.json';
 export { version } from '../../../data/version.json';
 
 export const apiConfig = {
-	apiAuthorityPart: 'api.zotero.org',
+	apiAuthorityPart: 'localhost:8080',
 	retry: 2,
 };
-export const websiteUrl = 'https://www.zotero.org/';
+export const websiteUrl = 'http://localhost:8080/';
 export const buyStorageUrl = websiteUrl + 'storage?ref=usb';
 export const stylesSourceUrl = 'https://www.zotero.org/styles-files/styles.json';
-export const streamingApiUrl = 'wss://stream.zotero.org/';
+export const streamingApiUrl = 'ws://localhost:8081/';
 export const translateUrl = 'location' in window ? window.location.origin : '';
 export const tinymceRoot = '/static/other/tinymce/';
 export const maxColoredTags = 9;
