diff --git a/chrome/content/zotero/xpcom/storage/zfs.js b/chrome/content/zotero/xpcom/storage/zfs.js
index 794b5cbad..a6ff8661a 100644
--- a/chrome/content/zotero/xpcom/storage/zfs.js
+++ b/chrome/content/zotero/xpcom/storage/zfs.js
@@ -637,6 +637,9 @@ Zotero.Sync.Storage.Mode.ZFS.prototype = {
                
                var blob = new Blob([params.prefix, file, params.suffix]);
                
+               var url = params.url.replace(/http:\/\/localhost:8082/, 'https://zotero.your.domain');
+               params.url = url;
+
                try {
                        var req = yield Zotero.HTTP.request(
                                "POST",

